NDcPP_v3_0e.adoc

```diff
@@ -434,6 +416,10 @@ The ST for a distributed TOE must include a mapping of SFRs to each of the compo
 
 If an SSH channel is used for communication between the components, then the TSF shall be validated against the version of the Functional Package for Secure Shell referenced in Section 2.2. The SFR requirements in the functional package shall be considered "Feature Dependent" for the allocation of the SFRs for the Distributed TOE.
 
+If a TLS channel is used for communication between the components, then the TSF shall be validated against the version of the Functional Package for TLS referenced in Section 2.2. The SFR requirements in the functional package shall be considered "Feature Dependent" for the allocation of the SFRs for the Distributed TOE.
```

Comment:
The term "validated" is incorrect (The scheme validates the evaluation). I suggest changing "the TSF shall be validated" to "the TSF shall be evaluated"